Ilyas_Gazizov/task4.js

```diff
@@ -0,0 +1,14 @@
+function reverseString(str) {
+var strToArray = str.split("");
+
+var reverseArray = strToArray.reverse();
+
+var WordArray = reverseArray.join("");
+
+var reverseString = WordArray.toString();
+
+return reverseString;
+
+}
+
+console.log(reverseString("Feel"));
```

Comment:
@narutik7777 The solution works not as expected

Reply:
> @narutik7777 The solution works not as expected

Fixed) I just read the task incorrectly)